rust_dev_preview/cross_service/rest_ses/src/work_item/mod.rs

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! WorkItem domain entity and error wrapper.
+pub mod collection;
+pub mod repository;
+pub mod work_item_archived;
+
+pub use work_item_archived::WorkItemArchived;
+
+use actix_web::{http::StatusCode, HttpResponse, ResponseError};
+use chrono::NaiveDate;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use thiserror::Error;
+use uuid::Uuid;
+
+/// A WorkItem, following the spec definition.
+/// Many fields use default values for both for core::Default and serde.
+/// `name` has an alias of `username`, to match the FE spec with the table definition.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct WorkItem {
+    #[serde(default = "uuid::Uuid::new_v4")]
+    idwork: Uuid,
```

Comment:
This might not matter, because I'm not sure the frontend uses it, but I think the FE expects `id`, so do you need to alias this as well?